### PR TITLE
fix 1B realm access issue

### PR DESCRIPTION
**Pull request by pwright**

### CODE REVIEW

File: walkthroughs/1B-protecting-applications-using-rh-sso/walkthrough.adoc [hunk 1]
```diff
@@ -44,7 +45,8 @@ SSO uses *Realms* to manage *Clients*, *Roles*, *Users* and *Groups*. A
 user belongs to and logs into a realm. Realms are isolated from one another and
 can only manage and authenticate the users that they control.
 
-A realm has already been created for this Solution Pattern and user.
+NOTE: The `walkthroughs` realm used in this Solution Pattern is shared with all users on the cluster. Do not use this realm for sensitive applications.
```
Comment (evanshortiss) [inline walkthroughs/1B-protecting-applications-using-rh-sso/walkthrough.adoc]: I'd change this to say "production applications" or "Only use this realm for Solution Pattern content.  Do not use this realm for production applications."

File: walkthroughs/1B-protecting-applications-using-rh-sso/walkthrough.adoc [hunk 2]
```diff
@@ -53,20 +55,20 @@ a *Realm*. The *Client* represents the application being secured and contains
...
 
 . Navigate to the link:{sso-realm-url}[SSO Realm, window="_blank"].
-. Enter the username `{user-username}` and password `password` if prompted.
-. Details for the `{user-username}` realm are displayed upon login success.
+. Enter the username `walkthroughs` and password `password` if prompted.
+. Details for the `walkthroughs` realm are displayed upon login success.
 . Select *Clients* from the menu on the left.
 . Click the *Create* button at the top of the list of clients to display the *Add Client* screen:
-.. Enter `order-entry-system` in the *Client ID* field.
+.. Enter `{user-username}` in the *Client ID* field.
```
Comment (evanshortiss) [inline walkthroughs/1B-protecting-applications-using-rh-sso/walkthrough.adoc]: I'd change this to `{user-username}-order-entry-system`. We might want them to add another. client in the future so having a better identifiers will be nice.

**Review by evanshortiss — CHANGES_REQUESTED** [review]
Looks good @pwright, just added a minor change request.

File: walkthroughs/1B-protecting-applications-using-rh-sso/walkthrough.adoc (hunk 1 above)
Comment (pwright) [reply walkthroughs/1B-protecting-applications-using-rh-sso/walkthrough.adoc]: went with 'production applications' option

File: walkthroughs/1B-protecting-applications-using-rh-sso/walkthrough.adoc (hunk 2 above)
Comment (pwright) [reply walkthroughs/1B-protecting-applications-using-rh-sso/walkthrough.adoc]: created a new attribute 'client-name' and set it as you suggested, that avoids confusion, spotted a wrong user of {user-username} as a result ;)

**Review by wei-lee — APPROVED** [review]
LGTM.

**Review by evanshortiss — COMMENTED** [review]
LGTM

**Pull request merged**